homeassistant/components/starlink/sensor.py

```diff
@@ -110,4 +94,24 @@ def native_value(self) -> StateType | datetime:
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=lambda data: now() - timedelta(seconds=data.status["uptime"]),
     ),
+    StarlinkSensorEntityDescription(
```

Comment:
It would be a lot easier to see whats going on here if the descriptions were kept in the same order as before